### PR TITLE
Feature/get headsail trim

### PR DESCRIPTION
**Pull request by LennartSchwier**
Implement first connection between backend and frontend to get the trim data for the head sail

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import React from 'react';
+import HeadSail from "./components/HeadSail";
 
 function App() {
   return (
     <div>
-      <p>saildog</p>
+      <HeadSail></HeadSail>
```
Comment (jamarob) [inline frontend/src/App.js:7]: You can use a self closing tag here

Comment (jamarob) [inline frontend/src/App.js:7]: You can use a void tag here `<Headsail/>`, looks nicer imho.

File: frontend/src/service/TrimDataService.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+const windSpeed = 12;
+const waveHeight = 2.4;
+const course = "beam_reach";
+const url = `/api/trim/headsail?wind=${windSpeed}&wave=${waveHeight}&course=${course}`
+
+export const getHeadSailTrim = () => {
+    axios.get(url).then(result => (result.data));
+}
```
Comment (jamarob) [inline frontend/src/service/TrimDataService.js:10]: You are missing a return statement here.

File: frontend/src/service/TrimDataService.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+const windSpeed = 12;
+const waveHeight = 2.4;
+const course = "beam_reach";
```
Comment (jamarob) [inline frontend/src/service/TrimDataService.js:5]: These will become parameters of `getHeadsailTrim` in the future?

**Review by jamarob — CHANGES_REQUESTED** [review]

File: frontend/src/components/HeadSail.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import {getHeadSailTrim} from "../service/TrimDataService";
+
+export default function HeadSail() {
+    return (
+        <div>
+            {getHeadSailTrim()}
```
Comment (jamarob) [inline frontend/src/components/HeadSail.js:7]: As `getHeadSailTrim` will, hopefully, return a promise you need to do the request with useEffect on component mount and set a state accordingly.

File: frontend/src/App.js (hunk 1 above)
Comment (LennartSchwier) [reply frontend/src/App.js:7]: Thx, for your review!

Unfortunately due to my branch structure only the latest frontend files were shown in this pull request. They only contain code to make the axios work. However, the more interesting changes are in the backend.

**Pull request merged**